content/en/docs/Dashboard/_index.md

```diff
@@ -45,7 +45,7 @@ kubectl apply --filename https://github.com/tektoncd/dashboard/releases/latest/d
 The latest release **may not be compatible** with your Tekton Pipelines
 installation, should you have an earlier release of Tekton Pipelines
 installed. For more compatibility information, see the
-[Tekton Dashboard Releases](https://github.com/tektoncd/dashboard/releases)
+[Tekton Dashboard Releases](https://github.com/tektoncd/dashboard#which-version-should-i-use)
 page.
```

Comment:
The link no longer points to the `Tekton Dashboard Releases` page, so we should probably change the text to match the new link.

Maybe something like the `Tekton Dashboard's "Which version should I use?" section`... or if that's too long, maybe the `Tekton Dashboard page`. (Or some other wording that makes more sense)

For reference, here are my suggested changes:

```suggestion
[Tekton Dashboard's "Which version should I use?"](https://github.com/tektoncd/dashboard#which-version-should-i-use)
section.
```

```suggestion
[Tekton Dashboard](https://github.com/tektoncd/dashboard#which-version-should-i-use)
page.
```